src/ray/raylet/node_manager.cc

```diff
@@ -2346,33 +2375,14 @@ void NodeManager::HandlePinObjectIDs(const rpc::PinObjectIDsRequest &request,
     object_ids.push_back(ObjectID::FromBinary(object_id_binary));
   }
   if (object_pinning_enabled_) {
```

Comment:
Should we remove this feature flag (separately)?

Reply:
Yeah, I think it's just used for LRU right now (and I guess if you wanted to turn off ref counting manually, but I don't think that should be recommended or exposed anymore...).